outlines/generate/api.py

```diff
@@ -516,9 +549,14 @@ def stream(
         max_tokens: Optional[int] = None,
         stop_at: Optional[Union[str, List[str]]] = None,
         seed: Optional[int] = None,
+        apply_chat_template: Optional[bool] = None,
         **model_specific_params,
     ):
         """Return a text generator from a prompt or a list of prompts."""
+        if apply_chat_template is None:
```

Comment:
To make this pythonic, we should have one obvious way of applying a chat template. IMO the argument should only be accepted in the constructor.